docs/changehistory/NextVersion.md

```diff
@@ -4,83 +4,50 @@ publish: false
 
 # NextVersion
 
-## Update guide
+With most major releases comes many new features and some breaking API changes. Several breaking changes are the removal of previously deprecated APIs. In other cases, some APIs have changed in ways that may require calling code to be adjusted. This document describes these in detail to help you upgrade.
+
+The first step in migrating to 3.0 is to try out the new [upgrade tool](#upgrade-guide) that makes it easy to react to all of the renames that took place during the 3.0 release.
+
+The table of contents created below will make it easier to navigate through the various changes within the release. It provides a summary of the more complicated or extensive changes made and details on why the change has been made.
+
+Table of Contents:
+
+- [Upgrade guide](#update-guide)
+- [New features](#new-features)
+  - [Display System](#display-system)
```

Comment:
Are these really the only new features we've introduced over the past six months, or is this document just missing lots of information?
(One of those options is pretty depressing...)
@NancyMcCallB @bsteinbk appui has a bunch of enhancements, right?

Reply:
There should be some more but a lot of the features we added are directly tied to breaking changes so we'll need to figure out how to extract some of that out to make it more clear.

Reply:
Perhaps it would be easier to just summarize the enhancements that motivated the breaking changes at the top.

Reply:
i.e., tell the reader what they get for upgrading to 3.0, in exchange for having to deal with breaking changes.

Reply:
We documented changes in the AppUi section, but we can separate the new features to be a "highlight reel" link under New Features. Is that what you have in mind, @pmconne?

Reply:
Yes, something like that - thanks.